pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -32,6 +34,7 @@ const (
 	UnauthenticatedGroup = "system:unauthenticated"
 	ClusterAdminGroup    = "system:cluster-admins"
 	ClusterReaderGroup   = "system:cluster-readers"
+	InfraGroup           = "system:serviceaccounts:openshift-infra"
```

Comment:
don't make a constant for this... namespace is configurable, and I don't think we're using this anywhere